documentation/get-started/tutorials/get-started-qfs.en.md

```diff
@@ -93,6 +93,11 @@ Layer
     layer like a WMS or to take a working copy of an unsupported format
     like an ECW or MrSID layer.
 
+!!! note
+    On iOS devices, It is only possible with a PNG raster with a PNW world file at the moment.
+
+    JPG raster are not supported.
```

Comment:
```suggestion
    On iOS devices, it is only possible with a PNG raster with a PNW world file at the moment.

    JPG raster are not supported.
```

Fix `It` should not start with capital letter.

Reply:
While I do not have iOS device and cannot test myself, I think it is not that only possible with PNG, but it's ONLY NOT possible with JPG.

Also, I think we can note that single file tiled file formats should be configured with non-jpg tiles too, for example `mbtiles`.

Reply:
@nirvn